src/Microsoft.Data.SqlClient/tests/ManualTests/Microsoft.Data.SqlClient.ManualTesting.Tests.csproj

```diff
@@ -241,6 +246,9 @@
     <ProjectReference Include="$(TestsPath)tools\Microsoft.DotNet.XUnitExtensions\Microsoft.DotNet.XUnitExtensions.csproj">
       <Name>Microsoft.DotNet.XUnitExtensions</Name>
     </ProjectReference>
+    <ProjectReference Include="..\tools\TDS\TDS.EndPoint\TDS.EndPoint.csproj" />
```

Comment:
Use `$(TestPath)` instead of `..\` ?

Reply:
sure, I can change that. it was auto added by vs.